packages/nest/src/exclusive-field.validator.test.ts

```diff
@@ -0,0 +1,61 @@
+import { describe, expect, it } from '@jest/globals';
+import 'reflect-metadata';
+import { plainToClass, Type } from 'class-transformer';
+import { Validator } from 'class-validator';
+import { ExclusiveFieldExists } from './exclusive-field.validator';
+
+class CharacterAbility {
+  strength?: number;
+  stealth?: number;
+  magic?: number;
+}
+
+class CharacterInput {
+  @Type(() => CharacterAbility)
+  @ExclusiveFieldExists()
+  ability: CharacterAbility;
+}
+
+const validator = new Validator();
+
+describe('ExclusiveFieldExists', () => {
+  it('should throw zero errors if only one of the fields exist', async () => {
+    expect.assertions(1);
```

Comment:
Shouldn't need these. Thrown exceptions are handled as expected. Probably were needed before support for promises was added.
```suggestion
```